addons/account_peppol/models/account_move.py

```diff
@@ -51,8 +51,12 @@ def _compute_peppol_move_state(self):
             else:
                 move.peppol_move_state = move.peppol_move_state
 
-    def _notify_by_email_prepare_rendering_context(self, message, **kwargs):
-        render_context = super()._notify_by_email_prepare_rendering_context(message, **kwargs)
+    def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, model_description=False,
```

Comment:
Why ? It makes everything longer when you don't need those parameters.

Reply:
Because all 6 other overrides do it like this and having this line prevents from calling `_notify_by_email_prepare_rendering_context(test_message, {})` which is actually used in test_mail.

Reply:
But is it a bad pratice to shorten long calls, or is it specific to here, ... ? Remember someone from framework PY telling me to do that (long time ago :D ) .

Reply:
This change is specific to here. The alternative it to change the definition in base (and all overrides) to something like you had here. At the end, the signatures must be compatible with each other - but I agree that `f(**kw)` is shorter and easier to override than having all parameters as position and keyword.